src/main/java/com/zepben/evolve/services/customer/CustomerServiceUtils.kt

```diff
@@ -42,12 +42,12 @@ import com.zepben.evolve.cim.iec61970.base.core.IdentifiedObject
 @JvmOverloads
 inline fun <R> whenCustomerServiceObject(
     identifiedObject: IdentifiedObject,
-    crossinline isCustomer: (Customer) -> R,
-    crossinline isCustomerAgreement: (CustomerAgreement) -> R,
-    crossinline isOrganisation: (Organisation) -> R,
-    crossinline isPricingStructure: (PricingStructure) -> R,
-    crossinline isTariff: (Tariff) -> R,
-    crossinline isOther: (IdentifiedObject) -> R = { idObj: IdentifiedObject ->
```

Comment:
I'm not sure if the crossinline on these was really necessary in the first place, but assumed there was a reason. did you run into problems or did you come to some conclusion it wasn't needed?

Reply:
After reading the documentation again we came to the conclusion it wasn't needed. The reason crossinline was used here was to make sure that the lambdas that get passed into this inlined function would also get inlined but after reading the doco we realised that is not what crossinline does and that behaviour of inlining lambdas is the default behaviour.